Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45522&attempt_id=772055a9-f2a2-4454-9863-424d123a8db7)

Attempted: distrho

<details><summary>Partial log (click to expand)</summary><p>

```
g++: error: unrecognized command line option '-msse'; did you mean '-fdse'?
g++: error: unrecognized command line option '-msse2'
make[3]: *** [drowaudio.make:87: intermediate/Release/dRowAudio.o] Error 1
make[2]: *** [Makefile:22: drowaudio] Error 2
make[2]: Leaving directory '/build/source/libs/drowaudio/build-drowaudio'
make[1]: *** [Makefile:4: all] Error 2
make[1]: Leaving directory '/build/source/libs/drowaudio'
make: *** [Makefile:104: lv2] Error 2
builder for '/nix/store/pcw04dlmjbr53ggc0xvp7g5ghyj4l5k1-distrho-ports-2018-04-16.drv' failed with exit code 2
error: build of '/nix/store/pcw04dlmjbr53ggc0xvp7g5ghyj4l5k1-distrho-ports-2018-04-16.drv' failed
```
</p></details>

--- a/pkgs/applications/audio/distrho/default.nix
+++ b/pkgs/applications/audio/distrho/default.nix
@@ -2,13 +2,14 @@
 , pkgconfig, premake3, xorg, ladspa-sdk }:
 
 stdenv.mkDerivation rec {
-  name = "distrho-ports-unstable-2018-01-01";
+  name = "distrho-ports-${version}";
+  version = "2018-04-16";
 
   src = fetchFromGitHub {
     owner = "DISTRHO";
     repo = "DISTRHO-Ports";
-    rev = "b200e7409aa9f6612c4d948932f6ce6f0a087f5a";
-    sha256 = "0672r0a9s6skzkxpjdraziwh5k8ivrfzvi4zcpkcg3zrv2hia2vz";
+    rev = version;
+    sha256 = "0l4zwl4mli8jzch32a1fh7c88r9q17xnkxsdw17ds5hadnxlk12v";
   };
 
   patchPhase = ''
@@ -46,6 +47,6 @@ stdenv.mkDerivation rec {
       TAL-Vocoder-2 TheFunction ThePilgrim Vex Wolpertinger
     '';
     maintainers = [ maintainers.goibhniu ];
-    platforms = platforms.linux;
+    platforms = [ "x86_64-linux" ];
   };
 }

